### ISSUE TITLE
Selection and deleting bug in IE

### ISSUE THREAD

**Issue by Nikcrysis** [facebook/draft-js#1185]
Draft-js 0.10.0
Internet Explorer 11

Sometimes when you try to select all text with mouse and delete with backspace it.. nothing happens.

<img src="https://i.gyazo.com/27168e7d794f0ecc11f43b6ab0fd915e.gif">

**Comment by marekolszewski** [comment]
I don't think this is IE specific. This happens for me on draftjs.org in Chrome, Safari, and Firefox on my Mac as well. It usually works once then stops working.

**Comment by stephenburgess8** [comment]
I have been dealing with this issue for several days. From what I can tell, the new selection is not triggering a selection event in DraftJS. Oddly, for a new text input instance, the selection seems to work fine. Then, after losing focus on returning to edit the text again, selecting the text fails to trigger the selection event.

The function I am passing as the `onChange` prop is _not_ being triggered by selecting text in IE11 only. Since this is happening, it seems to keep track of the old selection area and applies the change there. Typically this will be the first or last place in the text.

However my team is using DraftJS in a few different places with slightly different props and different CSS wrappers. The case where we're experiencing this is when the text is wrapped in container elements that use flex. We thought it may have something do with that but I have been unable to determine if that's the crucial difference. In the other way we are using DraftJS, we don't have this problem.

I would appreciate help on this as it has been frustrating me for several days. Thank you.

**Comment by stephenburgess8** [comment]
One further bit of clarification:

The first time after creating an draftjs editor on my page, I'm able to select and change styles appropriately. However if I lose focus in the editor and return to it, it's at that point that I'm unable to make any selection.

**Comment by florapdx** [comment]
I am also experiencing this bug, where I'm trying to select an atomic block and am losing the ability after the first go. On the 2nd+ selection try, selection anchor and start keys reflect the surrounding "buffer" divs, even though those don't appear to be selected in the editor (while my image, etc, does appear to be selected). Also using a flex wrapper, but don't see why this would affect only the 2nd+ try.

Will keep poking at it and will post if I find anything :)
